src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.storage.StorageHandler;
+import duke.ui.UserInterface;
+
+import static duke.parser.Parser.parseCommand;
+
+public class Duke {
+    private StorageHandler storageHandler;
+    private UserInterface ui;
+
+    public Duke(String filePath){
```

Comment:
need a space between `)` and `{` here.
Tip: find out how to correct these spacing issues on the IDE automatically!